UDPClientGUI/ViewModels/LoginViewModel.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Caliburn.Micro;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UDPCommGUI.ViewModels
+{
+    class LoginViewModel : Screen
+    {
+        private IEventAggregator _eventAggregator;
+        private string _username;
+        private string _password;
+
+        public LoginViewModel(IEventAggregator eventAggregator)
+        {
+            _eventAggregator = eventAggregator;
+            _eventAggregator.Subscribe(this);
+        }
+
+        public string Username
+        {
+            get => _username;
+            set
+            {
+                _username = value;
+                NotifyOfPropertyChange(nameof(Username));
+            }
```

Comment:
Something that Meha mentioned is that Caliburn.Micro has a "Set" method where you can provide the backing property.

It automatically notifies of property changes.

E.g. 
set => Set(...)